Dockerfile

```diff
@@ -124,7 +124,8 @@ RUN set -ex; \
 	echo 'podman:100000:65536' > /etc/subuid; \
 	echo 'podman:100000:65536' > /etc/subgid; \
 	ln -s /usr/local/bin/podman /usr/bin/docker; \
-	mkdir -p /podman/.local/share/containers/storage /var/lib/containers/storage; \
+	mkdir -p /podman/.local/share/containers/storage /var/lib/containers/storage /usr/share/containers; \
+	wget -O /usr/share/containers/seccomp.json https://src.fedoraproject.org/rpms/containers-common/raw/rawhide/f/seccomp.json ; \
```

Comment:
ah, so here you're providing the missing seccomp.json to fix https://github.com/containers/podman/issues/10556.
I ll apply this to the tar distribution as well.